packages/react-codemods/transforms/pf3-pf4.test.js

```diff
@@ -1,9 +1,29 @@
+import { EOL as SYSTEM_EOL } from 'os';
 import prettier from 'prettier';
 import { defineInlineTest, runInlineTest } from 'jscodeshift/dist/testUtils';
 import transform from './pf3-pf4';
 
-const prettierConfig = prettier.resolveConfig.sync(process.cwd());
-const pretty = src => prettier.format(src, { parser: 'babylon', ...prettierConfig });
+/**
```

Comment:
i am still seeing some tests fail locally in this file on Mac...happy to give you the output if needed.

Reply:
@priley86 The Travis build is failing too. Looks like that's related to the Prettier upgrade -- some files are missing some formatting it seems. See https://travis-ci.org/patternfly/patternfly-react/builds/486997667#L5198 -- are the issues you are running into the same ones?

I disabled the linting locally, since I get mostly noise due to #1256. Looks like it's the PF3 packages that are causing trouble, I should be able to lint them on my machine without problems.

I'll take a look!

Reply:
yes, i think you've captured it. Seems you just need a few more snapshot updates in react-codemods... `react-codemods` is not yet used in the wild (still a P.O.C.) so definitely not a risk right now. Just need to ensure tests pass. I will try to pull your branch again and build locally on Mac after Travis happy 😸